Beam/resources/assets/js/remplib.js

```diff
@@ -98,6 +106,15 @@ remplib = typeof(remplib) === 'undefined' ? {} : remplib;
                 if (typeof config.tracker.article.locked !== 'undefined') {
                     this.article.locked = config.tracker.article.locked;
                 }
+                if (typeof config.articleElementFn() !== 'undefined') {
```

Comment:
* Ked je nieco funkcia / callback, chceme to zavolat co "najneskor".
* Ak ta funkcia nie je definovana, cele to tu spadne a nic dalsie sa nevykona.